organizationMemberships.matchers.js

```diff
@@ -26,7 +26,7 @@ definePattern(
   target => {
     expect(target).to.be.anOrganizationMembership();
     expect(target).to.matchPattern({
-      acceptedAt: _.isISODateString,
+      acceptedAt: _.isDateOrAnISODateString,
```

Comment:
why or? are you doing this to account for the JSONification through the apis? if so I think it would be better if we used a helper like this:

```js
const { organizationMemberships } = await request(…);
organizationMemberships.forEach(organizationMembership => {
   parseDates(organizationMembership, 'accetedAt, 'rejectedAt', 'etc');
});
```

this is a lame limitation of JSON

Reply:
I copied the pattern from our feetPost.matchers: https://github.com/jlinclabs/jlinc-shared/blob/master/feedPosts.matchers.js#L32

If we start to parse dates like you suggest here, we should implement it across all a-api routes that get dates from the b-api. Perhaps a separate task?